### PR TITLE
General Refactor

### PR DESCRIPTION
**Pull request by lkriffell**
Optimize code
add strong params when needed
update styling

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -23,11 +23,11 @@
  div#borderimg{
    border-width: 50px;
    border-style: solid;
-   border-image: url(https://render.fineartamerica.com/images/images-profile-flow/400/images/artworkimages/mediumlarge/3/fenced-into-seattle-matt-mcdonald.jpg) 30 round;
+   border-image: url(https://media.giphy.com/media/3oEdv1GbekAakxXO8g/giphy.gif) 30 round;
```
Comment (sarynm12) [inline app/assets/stylesheets/application.scss:26]: new border looks rad

File: app/controllers/applications_controller.rb [hunk 2]
```diff
@@ -41,6 +28,28 @@ def submit
...
   end
 
+  def create_app_and_app_pets
+    new_app = Application.create!(application_params)
+    params[:checked_pets].each do |pet_id|
+      ApplicationsPets.create!({applications_id: new_app.id, pets_id: pet_id})
+    end
+  end
+
+  def check_params_for_empties
```
Comment (sarynm12) [inline app/controllers/applications_controller.rb:38]: Nice helper method

File: app/views/pets/edit.html.erb [hunk 3]
```diff
@@ -1,20 +1,7 @@
...
-  <%= text_field_tag :'pet[sex]', @pet.sex %><br>
-
-  <label>Image:</label><br>
-  <%= text_field_tag :'pet[image]', @pet.image %><br>
-
-  <%= hidden_field_tag :'pet[:id]', @pet.id %><br>
-
-  <%= submit_tag "Update Pet" %><br><br>
-<% end %>
+<%= render partial: 'form', locals: {
```
Comment (sarynm12) [inline app/views/pets/edit.html.erb:2]: so clean

File: app/views/pets/_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,43 @@
+<%= form_tag path, method: method do %>
```
Comment (fieldstyler) [inline app/views/pets/_form.html.erb:1]: Awesome job implementing partials!

File: spec/features/shelters/new_spec.rb [hunk 5]
```diff
@@ -17,12 +17,11 @@
     expect(page).to have_content("state")
     expect(page).to have_content("zip")
 
-    fill_in :'shelter[name]', with: "Alfredo's Adoption"
-    fill_in :'shelter[address]', with: 55837
-    fill_in :'shelter[city]', with: "Denver"
-    fill_in :'shelter[state]', with: "CO"
-    fill_in :'shelter[zip]', with: 34521
-
+    fill_in :name, with: "Alfredo's Adoption"
```
Comment (sarynm12) [inline spec/features/shelters/new_spec.rb:20]: looks great :)

**Review by sarynm12 — APPROVED** [review]

**Comment by fieldstyler** [comment]
Reviewed by: Saryn and Tyler

**Pull request merged**